### PR TITLE
feat(icons): add CopyIcon

### PR DESCRIPTION
**Pull request by hharnisc**
Adds the CopyIcon with the same SVG as the internal react component library

image

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-icons/__IconList.tsx [hunk 1]
```diff
@@ -3,19 +3,21 @@ import * as React from 'react';
 import {Text} from '@twilio-paste/text';
 import {Grid, StoryIcon} from './__StoryStyles';
 
+import {CopyIcon} from './src/CopyIcon';
 import {LoadingIcon} from './src/LoadingIcon';
 import {PlusIcon} from './src/PlusIcon';
 
 interface IconProps {
   title?: string;
-  decorative?: boolean;
```
Comment (hharnisc) [inline packages/paste-icons/__IconList.tsx]: I followed the instructions outlined here: https://github.com/twilio-labs/paste/blob/master/packages/paste-icons/README.md#how-can-i-contribute-icons

Interesting the tool removed the `?`

File: packages/paste-icons/src/CopyIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/**
+ * This file was automatically generated with @twilio-labs/svg-to-react
```
Comment (hharnisc) [inline packages/paste-icons/src/CopyIcon.tsx]: This was very smooth!

Comment (TheSisb) [reply packages/paste-icons/src/CopyIcon.tsx]: Thanks! :D

File: packages/paste-icons/__IconList.tsx (hunk 1 above)
Comment (TheSisb) [reply packages/paste-icons/__IconList.tsx]: Yup this change is good.  The `?` there was an oversight.

**Comment by serifluous** [comment]
Hey @hharnisc! I'm going to send you an updated SVG tonight. We're going to make the square in the icon more square.

Thanks for being proactive on this!

**Comment by hharnisc** [comment]
Sounds great @serifluous 🙌 I'll update the PR with the new SVG

**Comment by hharnisc** [comment]
@TheSisb I updated the `CopyIcon` with the approved design (thank you @serifluous 🙌 )

**Comment by hharnisc** [comment]
Updated Icon is bolder 💪 
image

**Pull request merged**